stories/Templates/GenericPage/genericpage.scss

```diff
@@ -47,6 +47,27 @@
   }
 }
 
-.progress-container {
-  margin-top: 3.813rem;
+.global-header {
+  + .progress-container {
+    margin-top: 4.6875rem;
+    position: fixed;
+    z-index: 1;
+
+    @include devicebreak(xlarge) {
+      margin-top: 7.1875rem;
+    }
+    .progress-top {
+      top: inherit;
+    }
+  }
+
+  + .progress-container + .grid-x + .pagehero-wrapper {
```

Comment:
@harishkumar-srijan We can write this inside `+ .progress-container {` on #51. 
Also why are we using too many adjacent selectors for `.pagehero-wrapper {`? 
Can you optimise/better use the above selector?

Reply:
done